web-ui/src/main/resources/catalog/components/admin/harvester/HarvesterDirective.js

```diff
@@ -66,16 +66,10 @@
               .success(function(data) {
                 scope.icons = data[0];
               });
-          // $http.get('admin.usergroups.list@json?id=' + 1)
-          //          .success(function(data) {
           $http.get('info?_content_type=json&type=languages', {cache: true})
               .success(function(data) {
                 scope.languages = data.language;
               });
-          $http.get('admin.group.list@json', {cache: true})
```

Comment:
To confirm if this should be part of the PR, seem not related.

Reply:
This API call request a service which does not exist so It looks it can be safely removed.